### DISCUSSION TITLE
Obsidian Hung UI Freeze

### DISCUSSION THREAD

**Discussion by kennygokh** [Vinzent03/obsidian-git#604] [Problems]
I have 2 obsidian vault, one of the vault encountered obsidian hung and UI freeze. If I disabled git plugin, it is working fine. Strangely, it is having issue on one of the vault, the other vault is working fine. Please advise.

**Comment by parteekxo** [comment]
Yup same issue, it isn't working and freezes the ui.

**Reply by aidanastridge** [reply]
Same – MacOS version upon startup. Other plugins: Kanban, Iconize, and Heatmap calendar. Deleted plugin through hidden files.

**Reply by aidanastridge** [reply]
I noticed iconize was causing problems.

**Comment by aidanastridge** [comment]
Sometimes this can be caused by other plugins interacting. I had a similar situation — merely having obsidian-git installed would spike Obsidian's CPU usage making it inoperable. I had a minimal amount of plugins: Kanban, LaTeX suite, and Iconize. For some reason, uninstalling Iconize would eliminate the problem.

This is on MacOS and Android.
